src/views/svg.js

```diff
@@ -7,6 +7,7 @@ const svg = async (ctx, page) => {
   const svg = await page.$eval('#container > svg', (e) =>
     // this is a work-around by @hat215, for more details:
     // https://github.com/mermaid-js/mermaid-live-editor/issues/26#issuecomment-667678228
+    //e.style.background = 'red';
```

Comment:
Is this for debugging? If yes, could you please remove it?

Reply:
👍